docs/hardware/devices/heltec-automation/index.mdx

```diff
@@ -32,3 +32,9 @@ Versatile ESP32-S3-based boards E-Ink development boards.
 | [Vision Master E213](./vision-master/?heltec=vision_master_e213) | ESP32-S3R8 | SX1262 | 2.4GHz b/g/n | 5.0 | NO  |
 | [Vision Master E290](./vision-master/?heltec=vision_master_e290) | ESP32-S3R8 | SX1262 | 2.4GHz b/g/n | 5.0 | NO  |
 | [Vision Master T190](./vision-master/?heltec=vision_master_t190) | ESP32-S3R8 | SX1262 | 2.4GHz b/g/n | 5.0 | NO  |
+
+## [Mesh Node](./mesh-node)
+
+| Name                                          | MCU      | Radio  | WiFi         | BT  | GPS |
+| --------------------------------------------- | :------- | :----- | :----------- | :-: | :-: |
+| [Mesh Node T114](./mesh-node/?mesh_node_t114) | nRF52840 | SX1262 | 2.4GHz b/g/n | 5.0 | YES |
```

Comment:
I think this is wrong, the Mesh Node does not have WiFi, right? Also GPS is 'optional'

Reply:
Sorry, it was careless. I've changed them.